tests/FakeItEasy.Specs/ObjectCallSpecs.cs

```diff
@@ -90,7 +88,7 @@ public IOutputWriter WriteArgumentValue(object value)
             public IDisposable Indent()
             {
                 // no tests require indenting yet
-                return unindenter;
+                return Unindenter.Instance;

```

Comment:
Why not just `A.Fake<IDisposable>()`? As your comment says, we don't need to indent at this point, and it would save the hassle of maintaining a class and an instance.